etc/kayobe/ansible/growroot.yml

```diff
@@ -77,7 +77,7 @@
           vars:
             pv: "{{ pvs.stdout | from_json }}"
             disk_tmp: "{{ pv.report[0].pv[0].pv_name[:-1] }}"
-            disk: "{{ disk_tmp[:-1] if disk_tmp[-1] == 'p' and disk_tmp[:4] == 'nvme' else disk_tmp }}"
+            disk: "{{ disk_tmp[:-1] if disk_tmp[-1] == 'p' and disk_tmp[:9] == '/dev/nvme' else disk_tmp }}"
```

Comment:
I ran the pvs command on my laptop and `pv_name` is `/dev/mapper/nvme0n1p3_crypt`. Presumably because it's an encrypted disk.

Reply:
But I would think we are unlikely to encounter this naming scheme on one of our deployments when using our overcloud DIB image.

Could we merge this and fix more issues in follow-up changes?